source/index.md

```
@@ -94,6 +115,6 @@ Report issues or ideas on Github 📢
 ## Contributing
 
 - Improving PRs or issues with the workflow catalog (only the catalog, not the workflows themselves) can be made [here](http://github.com/snakemake/snakemake-workflow-catalog)
-- Improving PRs or issues with the listed workflows can be made at the respective workflow repository (see individual [workflow pages](<docs/all_standardized_workflows>)).
-- Resources for creating new workflows can be found [here](<docs/snakemake>) or in more detail on the [Snakemake documentation](https://snakemake.readthedocs.io/en/stable/index.html)
-- New workflows will be [automatically added](<docs/catalog>) to the workflow catalog if they are contained in eligible Github repositories
+- Improving PRs or issues with the listed workflows can be made at the respective workflow repository (see individual [workflow pages](docs/all_standardized_workflows)).
+- Resources for creating new workflows can be found [here](docs/snakemake) or in more detail on the [Snakemake documentation](https://snakemake.readthedocs.io/en/stable/index.html)
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)